.ebextensions/00_nypl_provisioning.config

```diff
@@ -0,0 +1,15 @@
+files:
+  "/opt/elasticbeanstalk/hooks/appdeploy/pre/07_nypl_provsion.sh":
```

Comment:
typo in file name?

Reply:
No, that name is arbitrary. The EB system should just run any file in its deploy directories. We've tested this same config on few other EB apps already.

Reply:
That was me above. Didn't realize I was logged in under a different user.